docs/apis-tools/camunda-api-rest/tutorial.md

```diff
@@ -0,0 +1,273 @@
+
+  // Emit roles to output.
+  results.items.forEach((x) =>
+    console.log(`Role Name: ${x.name}; key: ${x.key}`)
+  );
+} catch (error) {
+  // Emit an error from the server.
+  console.error(error.message);
+}
+```
+
+8. In your terminal, run `node cli.js camunda8 list`.
+
+:::note
+This `list` command is connected to the `listRoles` function at the bottom of the `camunda-8.js` file, and executed by the `cli.js` file. While we will work with roles in this tutorial, you may add additional arguments depending on the API calls you would like to make.
+:::
+
+The existing roles (if any) will now output. If you have an invalid API name or action name, or no arguments provided, or improper/insufficient credentials configured, an error message will output as outlined in the `cli.js` file. If no action is provided, it will default to "assign" everywhere, except when unassigning a user.
+
+## Create a role (POST)
+
+To create a role, take the following steps:
+
+1. Outline your function, similar to the steps above:
+
+```javascript
+async function createRole([roleName]) {
+  const accessToken = await getAccessToken(authorizationConfiguration);
+  const camundaApiUrl = process.env.ZEEBE_BASE_URL;
+  const url = `${camundaApiUrl}/roles`;
+}
+```
+
+2. Configure the API call:
+
+```javascript
+const options = {
+  method: "POST",
+  url,
+  headers: {
+    Accept: "application/json",
+    Authorization: `Bearer ${accessToken}`,
+  },
+  data: {
+    name: roleName,
+  },
+};
+```
+
+3. Process the results from the API call. For example:
+
+```javascript
+try {
+  const response = await axios(options);
+
+  // Process the results from the API call.
+  const newRole = response.data;
+
+  // Emit new role to output.
+  console.log(`Role added! Name: ${roleName}. Key: ${newRole.roleKey}.`);
+} catch (error) {
+  // Emit an error from the server.
+  console.error(error.message);
+}
+```
+
+4. In your terminal, run `node cli.js camunda8 create <name>`, where `<name>` is the name of the new role.
+
+## Retrieve a role (GET)
+
+To retrieve a role, take the following steps:
+
+1. Outline your function, similar to the steps above:
+
+```javascript
+async function getRole([roleKey]) {
+  const accessToken = await getAccessToken(authorizationConfiguration);
+  const camundaApiUrl = process.env.ZEEBE_BASE_URL;
+  const url = `${camundaApiUrl}/roles/${roleKey}`;
+}
+```
+
+2. Configure the API call:
+
+```javascript
+const options = {
+  method: "GET",
+  url,
+  headers: {
+    Accept: "application/json",
+    Authorization: `Bearer ${accessToken}`,
+  },
+};
+```
+
+3. Process the results from the API call. For example:
+
+```javascript
+try {
+  // Call the endpoint.
+  const response = await axios(options);
+
+  // Process the results from the API call.
+  const results = response.data;
+
+  // Emit role to output.
+  console.log(
+    `Role Name: ${results.name}; Key: ${
+      results.key
+    }; Members: ${JSON.stringify(results.assignedMemberKeys)}`
```

Comment:
```suggestion
    };`
```

non-blocking but strong suggestion: 

Oops! I had intended to remove this field, because I think it [is going away](https://camunda.slack.com/archives/C07CTSU4Q9Z/p1733235204314249). Let me know if you'd like me to make the corresponding change in the camunda-api-tutorials repo.